setup.py

```diff
@@ -39,10 +39,12 @@ def get_version(rel_path):
         "Operating System :: OS Independent",
     ],
     packages=setuptools.find_packages(),
-    install_requires=["posix-ipc", "numpy"],
+    install_requires=["numpy", "python-dotenv"],
```

Comment:
There is maybe a problem with python-dotenv. If a user don't call the load_dotenv from his code, the package can't find the external .env file. 
So our library shouldn't load the .env file, but should call the env vierables and we don't need this import